ibm_mq/tests/README.md

```diff
@@ -0,0 +1,26 @@
+How to install pymqi in MacOs
+=============================
+To install pymqi on Mac, first set up the [IBM MQ toolkit for Mac OS][1].
+
+1. Download and extract the [toolkit][2].
+2. Set up the following environment variables:
+    ```
+    export MQ_INSTALLATION_PATH=~/IBM-MQ-Toolkit-Mac-x64-9.1.2.0
+    export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:$MQ_INSTALLATION_PATH/lib64
+    export PATH="${PATH}:~/IBM-MQ-Toolkit-Mac-x64-9.1.2.0/bin"
```

Comment:
```suggestion
    export PATH="${PATH}:$MQ_INSTALLATION_PATH/bin"
```

Reply:
> So does this integration support Mac OS ?

You can install a dev toolkit on mac (since February this year) but I don't think you can actually install IBM MQ on a mac